rust/src/lib/nm/nm_dbus/active_connection.rs

```diff
@@ -17,6 +17,7 @@ pub struct NmActiveConnection {
     pub iface_type: NmIfaceType,
     pub iface_name: String,
     pub state_flags: u32,
+    pub dev_obj_path: Option<String>,
```

Comment:
Is it really possible to have an ActiveConnection without a device? Otherwise, better not to use Option.